### ISSUE TITLE
config doesn't work with storage emulator (URI property for configuration should be full uri)

### ISSUE THREAD

**Issue by deepakverma** [Azure/azure-sdk-for-php#24]
Current implementation:
$config = Configuration::GetInstance();
$config->SetProperty(QueueConfiguration::ACCOUNT_NAME, 'test');
$config->SetProperty(QueueConfiguration::URI, 'queue.core.windows.net');

sdk implementation generates the uri by pre-pending the account name

Expected:
URI should be full url
$config->SetProperty(QueueConfiguration::URI, 'http://test.queue.core.windows.net');
as the current implementation won't work for
1. storage emulator where the uri is http://127.0.0.1
2. It should be consistent across languages.

Basically, the QueueRestProxy constructor should not append the account name to the uri.
